scss/tokens/_raw.scss

```diff
@@ -1,82 +1,407 @@
+$ouds-dimension-800: $ouds-dimension-base * 16 !default; // 64
+// $ouds-dimension-850: $ouds-dimension-base * 17 !default; // 68
+$ouds-dimension-900: $ouds-dimension-base * 18 !default; // 72
+$ouds-dimension-1000: $ouds-dimension-base * 20 !default; // 80
+$ouds-dimension-1200: $ouds-dimension-base * 24 !default; // 96
+$ouds-dimension-1400: $ouds-dimension-base * 28 !default; // 112
+$ouds-dimension-1600: $ouds-dimension-base * 32 !default; // 128
+$ouds-dimension-1800: $ouds-dimension-base * 36 !default; // 144
+// $ouds-dimension-2000: $ouds-dimension-base * 40 !default; // 160
+// $ouds-dimension-3000: $ouds-dimension-base * 80 !default; // 320
+// $ouds-dimension-4000: $ouds-dimension-base * 120 !default; // 480
+// $ouds-dimension-5000: $ouds-dimension-base * 140 !default; // 560
+$ouds-dimension-6000: $ouds-dimension-base * 160 !default; // 640
+$ouds-dimension-7000: $ouds-dimension-base * 180 !default; // 720
+$ouds-dimension-9000: $ouds-dimension-base * 220 !default; // 880
+$ouds-dimension-11000: $ouds-dimension-base * 260 !default; // 1040
+// scss-docs-end ouds-raw-dimension
+
 // Elevation
 
 // scss-docs-start ouds-raw-elevation
-$ouds-elevation-blur-0:   0 !default;
+$ouds-elevation-blur-0: 0 !default;
 // $ouds-elevation-blur-100: 1px !default;
 $ouds-elevation-blur-200: 2px !default;
 $ouds-elevation-blur-300: 3px !default;
 $ouds-elevation-blur-400: 4px !default;
 // $ouds-elevation-blur-500: 8px !default;
 $ouds-elevation-blur-600: 12px !default;
 // $ouds-elevation-blur-700: 20px !default;
+$ouds-elevation-spread-0: 0 !default;
+// $ouds-elevation-spread-300: 3px !default;
+$ouds-elevation-spread-n100: -1px !default;
+// $ouds-elevation-spread-n200: -2px !default;
+$ouds-elevation-spread-n300: -4px !default;
+// $ouds-elevation-spread-n400: -8px !default;
+$ouds-elevation-x-0: 0 !default;
+$ouds-elevation-y-0: 0 !default;
+$ouds-elevation-y-100: 1px !default;
+$ouds-elevation-y-200: 2px !default;
+$ouds-elevation-y-300: 4px !default;
+// $ouds-elevation-y-400: 8px !default;
+$ouds-elevation-y-500: 12px !default;
+// $ouds-elevation-y-600: 20px !default;
+// scss-docs-end ouds-raw-elevation
 
-$ouds-elevation-spread-n100:  -1px !default;
-// $ouds-elevation-spread-n200:  -2px !default;
-$ouds-elevation-spread-n300:  -4px !default;
-// $ouds-elevation-spread-n400:  -8px !default;
-$ouds-elevation-spread-0:     0 !default;
-// $ouds-elevation-spread-300:   3px !default;
+// Font
 
-$ouds-elevation-x-0:    0 !default;
+// scss-docs-start ouds-raw-font
+$ouds-font-family-brand-default: "Helvetica Neue" !default;
+// $ouds-font-family-brand-tv: "Helvetica Neue LT" !default;
+// $ouds-font-family-monospace-courier-new: "Courier New" !default;
+// $ouds-font-family-monospace-menlo: "Menlo" !default;
+// $ouds-font-family-monospace-monaco: "Monaco" !default;
+// $ouds-font-family-monospace-sf-mono: "SF Mono" !default;
+// $ouds-font-family-system-arial: "Arial" !default;
+// $ouds-font-family-system-helvetica: "Helvetica" !default;
```

Comment:
We will probably need some of those but aren't referenced in semantic, how should we proceed ? Should we manually uncomment the ones that we need ?

Reply:
Font is not handled right now in the tool. It will probably get rid of all the font family tokens.